core/src/main/scala/com/banno/kafka/consumer/ConsumerOps.scala

```diff
@@ -33,16 +33,39 @@ import com.banno.kafka._
 import fs2.concurrent.{Signal, SignallingRef}
 import org.typelevel.log4cats.slf4j.Slf4jLogger
 
-sealed trait SeekTo {
-  def apply[F[_]](consumer: ConsumerApi[F, ?, ?], partitions: Iterable[TopicPartition]): F[Unit]
-}
-case object SeekToBeginning extends SeekTo {
-  def apply[F[_]](consumer: ConsumerApi[F, ?, ?], partitions: Iterable[TopicPartition]): F[Unit] =
-    consumer.seekToBeginning(partitions)
-}
-case object SeekToEnd extends SeekTo {
-  def apply[F[_]](consumer: ConsumerApi[F, ?, ?], partitions: Iterable[TopicPartition]): F[Unit] =
-    consumer.seekToEnd(partitions)
+sealed trait SeekTo
+case object SeekToBeginning extends SeekTo
+case object SeekToEnd extends SeekTo
+case class SeekToTimestamps(timestamps: Map[TopicPartition, Long], default: SeekTo) extends SeekTo
+case class SeekToTimestamp(timestamp: Long, default: SeekTo) extends SeekTo
```

Comment:
suggestion: rename to `SeekToOneTimestamp` to visually distinguish from the other case---only an `s` of difference is difficult to discern.